app/assets/javascripts/application.js

```diff
@@ -17,6 +17,11 @@
 //= require numeral
 //= require miq_api
 //= require miq_angular_application
+//= require controllers/deployment/wizard-module
```

Comment:
@himdel ill try to remove it, i know we are requiring everything but we had a dependency problem, that is why we placed it upfront.